### ISSUE TITLE
Separate warning and info texts

### ISSUE THREAD

**Issue by MichaelEbert** [MichaelEbert/OblivionProgressTracker#312]
It looks weird for all the nirnroot to have a ⚠ icon next to them. We should differentiate between ⚠ for warnings and critical things, and ⓘ for helpful tips that are not critical, like location instructions.

**Comment by MeemawHustlin** [comment]
I agree. I was trying to think of a way to clean up the checklist stuff like, this seems like a good way of doing it.
